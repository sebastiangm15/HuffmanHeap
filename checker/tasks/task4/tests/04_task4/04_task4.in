15
3 STEY6
11 ST2R8
6 STQRY
2 ST4YV
3 STMMC
22 STL71
21 ST7JY
8 STKSR
14 STIAP
29 STZ1B
21 STC9N
38 STN37
1 STYME
15 STNSH
33 STCZK
6
STYME
STL71
STEY6
STN37
STKSR
STQRY
