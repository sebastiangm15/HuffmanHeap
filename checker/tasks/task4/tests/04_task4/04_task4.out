ST7JYSTC9NSTL71STIAPSTQRYSTKSRSTZ1BSTCZKSTNSHSTYMEST4YVSTEY6STMMCST2R8STN37
